StRoot/StFttClusterMaker/StFttClusterMaker.h

```diff
@@ -45,12 +51,18 @@ class StFttClusterMaker: public StMaker {
     float GetThresholdFor( StFttRawHit * hit ) { return 0.0;}
     bool PassTimeCut( StFttRawHit * hit );
 
+
+
     StEvent*             mEvent;
     StFttCollection*     mFttCollection;
     int                  mRunYear;
     bool                 mDebug;
     StFttDb*             mFttDb;
 
+    int mTimeCutMin  = -999;
+    int mTimeCutMax  =  999;
```

Comment:
You may want to introduce named constants at the class level which would also help to document your intention:

```cpp
const int defaultTimeCutMin = -999;
const int defaultTimeCutMax = 999;
```

There are standard named constants `INT_MIN` and `INT_MAX` defined by the language https://en.cppreference.com/w/cpp/types/climits They should be preferred over magic numbers.

Reply:
@plexoos Thanks for all of the suggestions. I have made the code (IMHO) much clearer both with your suggestions and a few other changes. 
please look at [aa226e4](https://github.com/star-bnl/star-sw/pull/686/commits/aa226e47c51be7cc84e9244929888f382c26deeb)